src/Components/EachPost/EachPost.jsx

```diff
@@ -21,45 +26,75 @@ export const EachPost = () => {
     })();
   }, []);
 
-  return posts.map((post) => {
-    return (
-      <div key={post._id} className={styles.post}>
-        <Link to={`/${post.username}`}>
-          <img
-            src={
-              post.profilePhoto
-                ? post.profilePhoto
-                : "https://cdn.pixabay.com/photo/2015/10/05/22/37/blank-profile-picture-973460_960_720.png"
-            }
-            alt="User Thumbnail"
-          />
-        </Link>
-        <div className={styles.post_user_details}>
-          <div className={styles.user_name}>
-            <Link
-              style={{ color: "var(--content-color)" }}
-              to={`/${post.username}`}
-            >
-              <h1>{post.name}</h1>
-            </Link>
-            <Link
-              style={{ color: "var(--content-color)" }}
-              to={`/${post.username}`}
-            >
-              <h2>@{post.username}</h2>
+  return loading ? (
+    <Loader />
+  ) : (
+    posts.map((post) => {
+      return (
+        <div key={post._id} className={styles.post}>
+          <Link to={`/${post.username}`}>
+            <img
+              src={
+                post.profilePhoto
+                  ? post.profilePhoto
+                  : "https://cdn.pixabay.com/photo/2015/10/05/22/37/blank-profile-picture-973460_960_720.png"
+              }
+              alt="User Thumbnail"
+            />
+          </Link>
+          <div className={styles.post_user_details}>
+            <div className={styles.user_name}>
+              <Link
+                style={{ color: "var(--content-color)" }}
+                to={`/${post.username}`}
+              >
+                <h1>{post.name}</h1>
+              </Link>
+              <Link
+                style={{ color: "var(--content-color)" }}
```

Comment:
Can remove the use of inline style from here.

Reply:
yes i will remove the inline styling thanks for pointing it out